OpenUpSA/wazimap-ng-ui
# my view panel

## Description
managing the sticky indicators via my view panel.

this PR contains https://github.com/OpenUpSA/wazimap-ng-ui/pull/674 and a lot of changes to it. this PR introduces automated tests for sticky indicators too. #674 is obsolete now and this PR should replace it. 

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-612

## How is it tested automatically?
added `__tests__/gui/my_view.feature`

## How should a reviewer test it locally
* confirm that adding and removing filters work correctly for both rich data and choropleth with my view panel
* confirm that the filter change in choropleth doesn't affect rich data and vice versa
* confirm that my view panel is working correctly with default filters and non-aggregatable filters
* confirm that selected filters are remained when switching to another geo
* confirm that this PR doesn't break filtering in point mapper filter dialog
* compare with production to make sure the values are still calculated & displayed correctly after filtering

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/209731252-31f66914-c03d-44aa-a643-53583a41e0b5.png)


## Changelog

### Added
* `js/elements/my_view` path

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
